samples/Playground/Playground.Shared/App.xaml.cs

```diff
@@ -190,6 +190,10 @@ protected async override void OnLaunched(LaunchActivatedEventArgs args)
 		if (logger.IsEnabled(LogLevel.Warning)) logger.LogWarningMessage("LogLevel:Warning");
 		if (logger.IsEnabled(LogLevel.Error)) logger.LogErrorMessage("LogLevel:Error");
 		if (logger.IsEnabled(LogLevel.Critical)) logger.LogCriticalMessage("LogLevel:Critical");
+
+		var windowServices = (_window.Content as FrameworkElement)!.FindServiceProvider();
+		var ts = windowServices.GetRequiredService<IThemeService>();
+		await ts.SetThemeAsync(ts.Theme);
```

Comment:
can we move this to an earlier state like line 174 in the initial navigate where we already have the Service Provider. This would also keep the UI from needing to refresh if we are updating the requested theme.

Will move this to the spec discussion but thinking to simplify this we should probably have a Restore/Initialize API... seems a bit dirty to have:
```cs
ts.SetThemeAsync(ts.Theme)
```